### PR TITLE
downloaded and setup the local images for expanding card

### PR DESCRIPTION
**Pull request by Deepak8717**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by atapas** [comment]
I would discard this PR. Let's create a fresh one.

- First pull the changes to your forked repo from this repo.
- Then make changes
- Push and Create PR.

**Comment by atapas** [comment]
@Deepak8717 I have tested it this way.

The changes from this branch is deployed here: https://react-play-op01h5grl-atapas.vercel.app/plays/expanding-cards

The initial load takes about 5-6 secs. Once loading is done, there after it is fast. I have confirmed the images are loading locally: 
image

The one already in production(https://www.reactplay.io/plays/expanding-cards) takes almost a similar of lesser time than the local image change. I have confirmed, previously the images were downloaded remotely: 
image

Hence the fix of this branch is not saving us from the initial load issue. You may want to check the issue from other perspective on how to load the images faster initially or give an explicit loading indicator so that users know it is loading. 

We can discuss here further.

**Comment by atapas** [comment]
> @Deepak8717 I have tested it this way.
> 
> The changes from this branch is deployed here: https://react-play-op01h5grl-atapas.vercel.app/plays/expanding-cards
> 
> The initial load takes about 5-6 secs. Once loading is done, there after it is fast. I have confirmed the images are loading locally: image
> 
> The one already in production(https://www.reactplay.io/plays/expanding-cards) takes almost a similar of lesser time than the local image change. I have confirmed, previously the images were downloaded remotely: image
> 
> Hence the fix of this branch is not saving us from the initial load issue. You may want to check the issue from other perspective on how to load the images faster initially or give an explicit loading indicator so that users know it is loading.
> 
> We can discuss here further.

@Deepak8717 The main issue is image size it 7MBs that too much. Please reduce and replace the images and push. That will fix the issue. Use PixResizer, it will bring images to KBs

image

File: src/plays/expanding-cards/Expanding-Card-Images.jsx [hunk 1]
```diff
@@ -1,24 +1,30 @@
+import titleOne from "./images/photo-1511028931355-082bb4781053.jpeg";
```
Comment (atapas) [inline src/plays/expanding-cards/Expanding-Card-Images.jsx:1]: The file sizes are huge. Please reduce them to KBs

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/plays/expanding-cards/Expanding-Card-Images.jsx (hunk 1 above)
Comment (Deepak8717) [reply src/plays/expanding-cards/Expanding-Card-Images.jsx:1]: Done, kindly review

**Comment by Deepak8717** [comment]
Hello @atapas , sorry after raised PR i realised its throwing error. I did test and it was running fine. however i believe while checking between branches it ask me to git pull and merge conflict and i belive thats where it has messed up again. please do look if it working ?

**Comment by atapas** [comment]
@Deepak8717 Can you please discard this one and raise a fresh PR?

**Pull request closed without merging**